### DISCUSSION TITLE
3/28(화) init(초기화)

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#38] [General]
오늘은 init에 대해서 공부해오는 것이 과제 입니다. init은 매우 중요합니다. 솔직히 모르면 나중에 이해하는데 시간이 너무 오래걸리기 때문에 초반에 잘 잡고 가는 것을 추천드립니다. 최근 제가 갠프와 개인 공부로 쪼끔 바빠져서 과제를 가끔 제출 못할 때가 있습니다. 다음날이면 가능하면 올리겠지만 못하면....ㅠ 하여간 요번주는 정처기 기간이기 때문에 오늘까지는 조금 큰과제를 냈지만 요번주는 간단한거 위주로 낼께요. 그리고 글이 길고 길지 않고는 중요하지 않습니다. 오늘 배운 내용을 이해했냐? 가 중요합니다. 😌

**Comment by Eunho0922** [comment]
넵
